backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -46,6 +85,15 @@ public static IQueryable<Entity.Project> GenerateQuery(this PimsContext context,
                 query = query.Where(p => p.StatusId == filter.StatusId);
             if (filter.TierLevelId.HasValue)
                 query = query.Where(p => p.TierLevelId == filter.TierLevelId);
+            if (filter.CreatedByMe.HasValue && filter.CreatedByMe.Value)
+            {
+                query = query.Where(p => p.CreatedById.Equals(user.GetUserId()));
+            }
+
+            if (filter.Active.HasValue && filter.Active.Value)
+            {
+                query = query.Where(p => p.Status.Name != "Denied");
```

Comment:
We'll need to rethink this - Tech Debt

Reply:
Are the project status order finalized? If yes, I can create extensions to check status